src/components/views/dialogs/RoomSettingsDialog.tsx

```diff
@@ -88,7 +116,7 @@ export default class RoomSettingsDialog extends React.Component<IProps, IState>
 
     private onRoomName = (): void => {
         this.setState({
-            roomName: MatrixClientPeg.get().getRoom(this.props.roomId)?.name ?? "",
+            roomName: this.state.room.name ?? "",
```

Comment:
This may be overtaken by my other comment in this file, but could we omit the `?? ""` here? It looks to me like a `Room` always has a `.name` string